flowers/trainer/task.py

```diff
@@ -33,6 +33,10 @@
 import tensorflow as tf
 from tensorflow.python.lib.io import file_io
 
+# Path to a default checkpoint file for the Inception graph.
+DEFAULT_INCEPTION_CHECKPOINT = (
+    'gs://cloud-ml-data/img/flower_photos/inception_v3_2016_08_28.ckpt')
```

Comment:
use `gs://cloud-samples-data/ml-engine/flowers/inception_v3_2016_08_28.ckpt` instead.  (the cloud-ml-data bucket is not maintained)